### PR TITLE
Add more get started docs to the program IDA

### PR DESCRIPTION
**Pull request by schenedx**
@jimabramson @rlucioni I have made some changes to the program IDA's getting started documentation to help the next person setup their Programs IDA on their local machine.

### CODE REVIEW

File: docs/getting_started.rst [hunk 1]
```diff
@@ -20,7 +20,7 @@ Local/Private Settings
 ----------------------
 When developing locally, it may be useful to have settings overrides that you do not wish to commit to the repository.
 If you need such overrides, create a file :file:`programs/settings/private.py`. This file's values are
-read by :file:`programs/settings/local.py`, but ignored by Git.
+read by :file:`programs/settings/local.py`, but ignored by Git. We do have an example `programs/settings/private.py.example` which you can follow
```
Comment (rlucioni) [inline docs/getting_started.rst]: "For an example, see `programs/settings/private.py.example`."

File: docs/getting_started.rst [hunk 2]
```diff
@@ -90,6 +90,19 @@ If you change the Programs schema, please update the fixture. You can do so by i
 
 .. _Django's migrate command: https://docs.djangoproject.com/en/1.8/ref/django-admin/#django-admin-migrate
 
+Collect Statics
```
Comment (rlucioni) [inline docs/getting_started.rst]: Asset Pipeline

File: docs/getting_started.rst [hunk 3]
```diff
@@ -90,6 +90,19 @@ If you change the Programs schema, please update the fixture. You can do so by i
...
+---------------
+
+The Programs IDA current do have UI components you can use and see. The static files needs to be processed and collected before you can use these UI comp.
```
Comment (rlucioni) [inline docs/getting_started.rst]: Static files for Programs are managed using gulp-sass, RequireJS and r.js, Django's `collectstatic`, and django-compressor. gulp-sass compiles Sass to CSS. RequireJS and r.js are used to manage JavaScript dependencies and optimize CSS. django-compressor handles naming files to facilitate cache busting during deployment.

When deploying to production, call `make static` to compile all static assets and move them to the proper location to be served. All tools should operate seamlessly in a local development environment.

File: docs/getting_started.rst [hunk 4]
```diff
@@ -90,6 +90,19 @@ If you change the Programs schema, please update the fixture. You can do so by i
...
+Execute the following command to get your statics setup:
```
Comment (rlucioni) [inline docs/getting_started.rst]: Execute the following command to run the asset pipeline locally:

File: docs/getting_started.rst [hunk 5]
```diff
@@ -90,6 +90,19 @@ If you change the Programs schema, please update the fixture. You can do so by i
...
+
+.. code-block:: bash
+
+    $ DJANGO_SETTINGS_MODULE=programs.settings.test make static
+
+The command above will prep and collect all the javascript files
```
Comment (rlucioni) [inline docs/getting_started.rst]: I'd remove this line, I think it's clear from the above what's going on.

File: docs/getting_started.rst [hunk 6]
```diff
@@ -90,6 +90,19 @@ If you change the Programs schema, please update the fixture. You can do so by i
...
+
+If you do not want the javascript being uglified, go to file `build.js` from the project root and modify the value of key `optimize`
```
Comment (rlucioni) [inline docs/getting_started.rst]: You can control JavaScript optimization from the project's `build.js` file by modifying the value of the `optimize` key.

**Comment by schenedx** [comment]
@rlucioni I took in all your feedbacks. Please review again?

File: docs/getting_started.rst [hunk 7]
```diff
@@ -20,8 +20,8 @@ Local/Private Settings
 ----------------------
 When developing locally, it may be useful to have settings overrides that you do not wish to commit to the repository.
 If you need such overrides, create a file :file:`programs/settings/private.py`. This file's values are
-read by :file:`programs/settings/local.py`, but ignored by Git.
-
+read by :file:`programs/settings/local.py`, but ignored by Git. 
+For an example, see programs/settings/private.py.example
```
Comment (rlucioni) [inline docs/getting_started.rst]: Nit: this is a sentence, so it should end with a period.

**Comment by rlucioni** [comment]
Nit aside, :+1: . Thanks for doing this.

**Comment by jimabramson** [comment]
great thanks @schenedx :+1:

**Pull request merged**